de.tu_bs.cs.isf.e4cf.core/src/de/tu_bs/cs/isf/e4cf/core/db/test/Test.java

```diff
@@ -17,23 +17,25 @@ public static void main(String[] args) throws SQLException, IOException {
 		final String DATABASENAME = "TestDatenbank";
 		final String TABLEENAME = "testTabelle";

```

Comment:
Vielleicht macht es sinn, diese in einen String Table auszulagern wenn Datenbankname feststeht?

Reply:
Nein. Es ergibt keinen Sinn, dies zu machen. Erstmal ist diese Datei für uns angelegt. Da wird schnell getestet, ob eine Methode bzsw. ein Workflow richtig implementiert wurde oder nicht. Man sieht sofort, dass die Datei nicht schön organisiert geschrieben wurde wie die anderen Dateien und so viele kommentierte Methodenaufrufe hat. Diese Datei wird im letzten Release nicht mit veröffentlicht sondern nur JUnitTests (eine ist schon angelegt). 
In der Softwareentwicklung ist es so, wenn man ungeänderte Strings im Laufe des Programms benutzt, sollten diese als Konstanten angelegt werden (das ist sogar eine Konvention). String Table ("Array") ist in dem Fall völlig unnötig. (und wenn dann heute eher ArrayList).
Die wichtigsten Funktionalitäten befinden sich im Package db.
Danke für die Anregung.